app/api/export-pdf/route.ts

```
@@ -0,0 +1,497 @@
+
+        // Function to render a document element
+        const renderElement = (element: any): void => {
+            if (!element) return;
+
+            checkAndAddNewPage();
+
+            switch (element.type) {
+                case 'judul':
+                    doc.setFontSize(16);
+                    doc.setFont("helvetica", "bold");
+                    const titleText = element.content || '';
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)